charts/flagsmith/templates/deployment-task-processor.yaml

```
@@ -44,7 +44,10 @@ spec:
       {{- if .Values.taskProcessor.schedulerName }}
       schedulerName: "{{ .Values.taskProcessor.schedulerName }}"
       {{- end }}
-      {{- if .Values.api.image.imagePullSecrets }}
+      {{- if (.Values.taskProcessor.image).imagePullSecrets }}
```

```suggestion
      {{- if .Values.taskProcessor.image.imagePullSecrets }}
```
because I think equivalent?